playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,67 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+After the delegation metadata is added and signed, the delegation
+keyholder should open a PR against the ceremony branch.
+The name of the PR MUST be `feat/add-delegation for
+<delegation-name>`. When creating the delegation, a  `target-meta`
+file has to be provided that lists the targets, similar when adding
+the top level targets.
+
+As part of running the `add-delegaton` command, a POP (proof of
+possession) has to be generated too. The computed POP should be stored
+in `${REPO}/staged/${FORK_POINT}.sig`, where the fork point is the
+fork point from `main` and the ceremony branch. This fork point is
+also used as the nonce when computing the pop.
+
+The delegation keyholder would run these commands (on a branch based
+on the ceremony branch):
+
+Create the delegation metadata
+```shell
+$ ./tuf add-delegation -name ${DELEGATION_NAME} \
+      -public-key ${PUB_KEY_REF} \
+      -target-meta delegate-meta.yaml \
+      -repository repository
+```
+
+```shell
+$ ./tuf sign \
+      -roles ${DELEGATION_NAME} \
+      -key ${KEY_REF} \
+      -repository ${REPO}
+```
+
+```shell
+$ FORK_POINT=$(git merge-base --fork-point origin/main "${BRANCH}") \
+      ./tuf key-pop-sign \
+      -key ${KEY_REF} \
+      -challenge ${DELEGATION_NAME} \
+      -nonce ${FORK_POINT} > ${REPO}/staged/${FORK_POINT}.sig
```

Comment:
Any risk of nonce reuse for multiple delegations signed from the same fork point? Any reason this isn't a random value that we store somewhere?

Reply:
This would be an issue only if we add multiple delegations at the same signing ceremony. We could add a step where the "ceremony orchestrator" adds a nonce in e.g. the `repository/staged` directory, with the name of the role, that could be used. I'll create a tracking issue for this, as I don't think it's a blocker now.

Also, I'm not sure if there is a risk in reusing the nonce as the challenge (role name) is also part of the POP, and the challenge is unique. But let's discuss this further in the issue.

Reply:
https://github.com/sigstore/root-signing/issues/729